packages/core/src/components/ui/Breadcrumb/Breadcrumb.tsx

```diff
@@ -4,28 +4,26 @@ import { memo } from 'react'
 
 import Link from 'src/components/ui/Link'
 
-type ItemElement = {
-  item: string
-  name: string
-  position: number
-}
 export interface BreadcrumbProps extends UIBreadcrumbProps {
-  /**
-   * Array of ItemElement that represents each breadcrumb item.
-   */
-  breadcrumbList: ItemElement[]
-}
-
-interface BaseBreadcrumbProps extends BreadcrumbProps {
-  isDesktop?: boolean
+  icon: string
+  alt: string
 }
 
-const Breadcrumb = ({ breadcrumbList, ...otherProps }: BaseBreadcrumbProps) => (
+const Breadcrumb = ({
+  icon = 'Home',
+  alt = 'Go to homepage',
+  ...otherProps
+}: BreadcrumbProps) => (
   <UIBreadcrumb
-    breadcrumbList={breadcrumbList}
     homeLink={
-      <Link aria-label="Go to homepage" href="/" prefetch={false}>
-        <Icon name="House" width={18} height={18} weight="bold" />
+      <Link
+        data-fs-breadcrumb-link
+        data-fs-breadcrumb-link-home
+        aria-label={`${alt}`}
```

Comment:
The `alt` incoming from CMS is a string, maybe we don't need this concatenation here, WDYT?

Reply:
nice catch! done [here](https://github.com/vtex/faststore/pull/1747/commits/8178118bfa69ddcb39e0eefc106271853b662b87).